HACKING.org

```diff
@@ -23,16 +23,26 @@ Git is used to manage the script's state:
 - If a supported OCaml version does not have CI support, run test suite locally.
 
 - In =CHANGES.md=, change =(unreleased)= with the current version (=a.b.c=)
-  and date. Commit the result.
+  and date. In =README.md=, update the recommended version to use in
+  =.ocamlformat= files. Commit the changes.
 
 - Tag
   =dune-release tag=
 
 - Check that release binary has a release version number:
   =dune build @install && dune install --prefix _install && ./_install/bin/ocamlformat --version && rm -rf _install= should output just =a.b.c=
 
+- Linting
+  =dune-release lint=
+
+- Amend the release commit if necessary.
+
+- Push the changes.
```

Comment:
The command needs to be `git push --tags`, otherwise the tag won't be pushed.
It is not necessary to push the tag now because `dune-release` will do it if necessary, however it can fail because it can only guesses the URIs and might be wrong.

Reply:
done